src/ApiService/ApiService/onefuzzlib/orm/EntityConverter.cs

```diff
@@ -205,6 +205,11 @@ private EntityInfo GetEntityInfo<T>()
 
                 tableEntity.Add(prop.columnName, string.Join(",", values));
             }
+            else if (prop.type == typeof(Container))
```

Comment:
don't couple the entity converter with the enties themselves.
instead, create jsonconverter on the entity type

Reply:
`BaseEventConverter` is one example of jsonConverter. In your case you can create a converter and apply it to the `Container` type

Reply:
also this fix onlyworks when the entity is saved to the db but not for when we read a json string